Maths/MatrixUtilites.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * @author Joseph Acevedo
+ * @file MatrixUtilities
+ * @purpose This file provides utilities for math with nxm matrices. The 
+ * following are provided methods: 
+ * matrixMultiply(double[][] mat_a, double[][] mat_b)
+ * 		Takes in two matrices of doubles and attempts to multiply them if they
+ * 		are of compatible sizes. If they are not compatible then an
+ * 		IllegalArgumentException is thrown
+ * matrixTranspose(double[][] mat)
+ * 		Transposes the given matrix for any nxm matrix and returns the
+ * 		resulting mxn matrix
+ */
+
+public class MatrixUtilites {
+	
+	/**
+	 * Multiples 2 matrices together given their dimensions are compatible
+	 * @param mat_a A 2-Dimensional matrix to be multiplied
+	 * @param mat_b A 2-Dimensional matrix to be multiplied
+	 * @return The product of the two matrices if compatible dimensions, 
+	 * 			otherwise an IllegalArgumentException is thrown
+	 */
+	public static double[][] matrixMultiply2D(double[][] mat_a, double[][] mat_b)
```

Comment:
Please use java conventions for nomenclature.

```suggestion
	public static double[][] matrixMultiply2D(double[][] matA, double[][] matB)
```